crates/test-utils/src/transaction.rs

```diff
@@ -320,9 +320,21 @@ pub async fn transfer_coin(
     let client = context.get_client().await.unwrap();
     let object_refs = client
         .read_api()
-        .get_objects_owned_by_address(sender)
+        .get_objects_owned_by_address(
+            sender,
+            Some(SuiObjectDataOptions::full_content()),
+            None,
+            None,
+        )
         .await?;
-    let object_to_send = object_refs.get(1).unwrap().object_id;
```

Comment:
Is there an easier way to parse this from the `SuiObjectResponse` ?

Reply:
I think you can add a util method in `impl SuiObjectResponse`